src/settings/components/utils.ts

```diff
@@ -496,13 +496,32 @@ export function defaultAsFirstOrderIdentities(identities: Array<Identity>): Arra
 	return result;
 }
 
+/**
+ * Return the requested attribute value as an array if present, or an empty array otherwise.
+ *
+ * @param settings
+ * @param key
+ * @returns
+ */
+export function getAttributeIfPresent(settings: AccountSettings, key: string): Array<string> {
+	const attributeValue = settings.attrs[key];
+	if (attributeValue) {
+		if (isArray(attributeValue)) {
+			return [...(attributeValue as string[])];
```

Comment:
The assertion is wrong. If the attribute is an array of number, the returned value will be an array of number, while the type declare it will always be an array of strings